distbench_netutils.cc

```diff
@@ -107,12 +107,16 @@ absl::StatusOr<DeviceIpAddress> GetBestAddress(std::string_view netdev,
   int score = -1;
   DeviceIpAddress best_match;
   for (const auto& address : all_addresses) {
-    if (address.isLinkLocal() || address.isLoopback()) {
+    if (address.isLinkLocal()) {
```

Comment:
Is there a way to specify that the IPv6 preference? Won't this logic return if there are only IPv6 addresses that are supported? Seems like IPv6 will be stuck at a score of -1?